packages/core/admin/admin/src/translations/en.json

```diff
@@ -181,15 +181,16 @@
   "Settings.permissions.auditLogs.action": "Action",
   "Settings.permissions.auditLogs.date": "Date",
   "Settings.permissions.auditLogs.user": "User",
+  "Settings.permissions.auditLogs.user.fullname": "{firstname} {lastname}",
   "Settings.permissions.auditLogs.userId": "User ID",
   "Settings.permissions.auditLogs.details": "Log Details",
   "Settings.permissions.auditLogs.payload": "Payload",
   "Settings.permissions.auditLogs.listview.header.subtitle": "Logs of all the activities that happened in your environment",
-  "Settings.permissions.auditLogs.entry.create": "Create entry ({model})",
-  "Settings.permissions.auditLogs.entry.update": "Update entry ({model})",
-  "Settings.permissions.auditLogs.entry.delete": "Delete entry ({model})",
-  "Settings.permissions.auditLogs.entry.publish": "Publish entry ({model})",
-  "Settings.permissions.auditLogs.entry.unpublish": "Unpublish entry ({model})",
+  "Settings.permissions.auditLogs.entry.create": "Create entry {model, select, undefined {} other {({model})}}",
+  "Settings.permissions.auditLogs.entry.update": "Update entry {model, select, undefined {} other {({model})}}",
+  "Settings.permissions.auditLogs.entry.delete": "Delete entry {model, select, undefined {} other {({model})}}",
+  "Settings.permissions.auditLogs.entry.publish": "Publish entry",
+  "Settings.permissions.auditLogs.entry.unpublish": "Unpublish entry",
```

Comment:
Why do you not show the model anymore here?

Reply:
I think I do as long as model is not undefined

Reply:
oh woops I just understood 🤣 , sorry I'll update